packages/lesswrong/components/form-components/ImageUpload.tsx

```diff
@@ -29,28 +30,31 @@ const styles = (theme: ThemeType): JssStyles => ({
   imageIcon: {
     marginRight: theme.spacing.unit
   },
+  removeButton: {
+    marginLeft: 10
+  }
 });
 
 const cloudinaryArgsByImageType = {
   gridImageId: {
-    min_image_height: 80,
-    min_image_width: 203,
-    cropping_aspect_ratio: 2.5375,
-    upload_preset: cloudinaryUploadPresetGridImageSetting.get(),
+    minImageHeight: 80,
+    minImageWidth: 203,
+    croppingAspectRatio: 2.5375,
+    uploadPreset: cloudinaryUploadPresetGridImageSetting.get(),
   },
   bannerImageId: {
-    min_image_height: 380,
-    min_image_width: 1600,
-    cropping_aspect_ratio: 2.5375,
-    cropping_default_selection_ratio: 3,
-    upload_preset: cloudinaryUploadPresetBannerSetting.get(),
+    minImageHeight: 200,
+    minImageWidth: 600,
```

Comment:
I do think 1600px is wider than necessary, so I figured I'd just let people upload smaller images if they want.